project1/styles.css

```diff
@@ -0,0 +1,83 @@
+body {
+    color: rgb(85, 85, 85);
+    background-color: rgb(230, 230, 230);
+    font-family: 'Cormorant Garamond', serif; 
+    
+ 
+     
+}
+
+.wrapper {
+    margin-top: 10px;
+    margin-right: auto;
+    margin-bottom: 10px;
+    margin-left: auto; 
+    text-align: center;
+    
+}
+
+h1 {
```

Comment:
Look for places where styles are shared by elements and combine them, like:
```css
h1, h2 {
    line-height: 2;
    text-align: center;
}
```
Then you can delete those declarations on the individual `h1` and `h2` below.